File: C++/jackolanternjuxtaposition.cpp
/*
 *	Problem: "Jack-O'-Lantern Juxtaposition"
 */
 
#include <iostream>

using namespace std;

/*-----main-----*/
int main(void) {

	int numEyes, numNose, numMouth;
	cin >> numEyes >> numNose >> numMouth;
	
	int result = numEyes * numNose * numMouth;
	cout << result;
	
	return 0;
} // end main

File: Ruby/jackolanternjuxtaposition.rb
=begin
 *
 *	Problem:		Jack-O'-Lantern Juxtaposition
 *	Problem ID:	jackolanternjuxtaposition
 *	url:				https://open.kattis.com/problems/jackolanternjuxtaposition
 *	Solution:		Ruby
 *
=end

class Solver

	def input_to_int(input)
		input_array = input.chomp.split(" ")
		input_int = input_array.map(&:to_i)
		return input_int
	end

	def solve
		num_eye, num_nose, num_mouth = input_to_int(gets)
		result = num_eye * num_nose * num_mouth
		puts(result)
	end
	
end

solver = Solver.new
solver.solve
